src/main/java/pvs/app/dto/GitlabCommitDTO.java

```diff
@@ -0,0 +1,58 @@
+package pvs.app.dto;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.Data;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Optional;
+import java.util.TimeZone;
```

Comment:
Suggest optimize the imports if decided to remove the `getAddSecondsToCommittedDateISOString` function.

```suggestion
import java.util.Date;
import java.util.Optional;
```